App/components/CigarettesText/CigarettesText.tsx

```diff
@@ -82,18 +82,23 @@ export function CigarettesText(props: CigaretteTextProps): React.ReactElement {
 	// Round to 1 decimal
 	const cigarettesRounded = Math.round(cigarettes * 10) / 10;
 
+	const nestingTrans = {
+		swear_word: t('swear_word', { context: swearWordId.toString() }),
+		cigarette_count: t('cigarette_count', { count: cigarettesRounded }),
+		frequency: t('frequency', { context: frequency || 'daily' })
+	}
+
 	return (
 		<Text style={[theme.shitText, style]} {...rest}>
 			<Trans
 				i18nKey="cigarette_report"
-				values={{ id: swearWordId, frequency: frequency || 'daily', cig_count: cigarettesRounded }}
+				values={{ ...nestingTrans }}
 				t={t}
 			>
-				$t(swear_word, {'{'} "context": "{'{{id}}'}" {'}'})! You smoke{'\n'}
+				{'{{swear_word}}'}! You smoke{'\n'}
 				<Text style={styles.cigarettesCount}>
-					$t(cigarette_count, {'{'} "count": "{'{{cig_count}}'}" {'}'})
-				</Text>{' '}
-				$t(frequency, {'{'} "context": "{'{{frequency}}'}" {'}'}).{/* TODO question about this frequency */}
+					{'{{cigarette_count}}'}
+				</Text> {'{{frequency}}'}.
```

Comment:
This helps both of translators and devs have a more comfortable view. Translators don't have to concern about nesting translation. Devs get less mess code. However, there is a price for it. The text is not sync in while changing frequency level. For example, if I change from `daily` to `weekly`, it will happen like this:

1. The `{{frequency}}` will shift from daily to weekly first.
2. Next is `{{cigarette_count}}` and `{{swear_word}}` at the same time.

I think the cause is the Internet connect. With `{{cigarette_count}}` and `{{swear_word}}`, the app takes time to download data and process it.